src/log.c

```diff
@@ -11,7 +11,8 @@
 #include "util.h"
 #include "config.h"
 
-bool verbose;
+bool verbose = false;
+
```

Comment:
Do we actually need `verbose` declared here and `extern` to `log.h`?

Reply:
If we want to keep both GCC and Clang happy, yes.